packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -354,16 +354,17 @@ export class MultiSinkLogger extends TelemetryLogger {
  * Helper class to log performance events
  */
 export class PerformanceEvent {
-    public static start(logger: ITelemetryLogger, event: ITelemetryGenericEvent) {
-        return new PerformanceEvent(logger, event);
+    public static start(logger: ITelemetryLogger, event: ITelemetryGenericEvent, reportStartEvent = true) {
```

Comment:
maybe a new static function? like track or something

Reply:
maybe change the param name to be something like `exludeEvents?:{start?:true,end?:true,cancel?:true,cancel:?}`

---
In reply to: [586891559](https://github.com/microsoft/FluidFramework/pull/5363#discussion_r586891559) [](ancestors = 586891559)

Reply:
Good point, changed, please take a look!